src/plugins/alexaSkill.js

```diff
@@ -0,0 +1,28 @@
+import idx from 'idx';
+import flatten from 'flat';
+
+import { pluginName } from '../util/constants';
+
+const type = 'alexaSkill';
+
+function eventType(e = {}) {
+  return idx(e, o => o.session.attributes) &&
+    idx(e, o => o.session.user) &&
+    idx(e, o => o.context.System) &&
+    idx(e, o => o.request.requestId)
+    ? type
+    : false;
+}
+
+function plugin(event, log) {
+  const obj = flatten(event);
+  Object.keys(obj).forEach(key => {
```

Comment:
so we're logging all of the keys?

Reply:
It looks like keys we *don't* want capture (at least):

```@iopipe/event-info.alexaSkill.context.System.apiAccessToken
@iopipe/event-info.alexaSkill.context.System.user.accessToken
@iopipe/event-info.alexaSkill.context.System.user.permissions.consentToken
```

Reply:
I can blacklist those, thanks @ewindisch makes it easy to update!